libraries/botbuilder-dialogs-adaptive/src/selectors/firstSelector.ts

```diff
@@ -22,11 +22,21 @@ export class FirstSelector implements TriggerSelector {
      */
     public parser: ExpressionParserInterface = new ExpressionParser()
 
+    /**
+     * Initialize the selector with the set of rules.
+     * @param conditionals Possible rules to match.
+     * @param evaluate True if rules should be evaluated on select.
```

Comment:
```suggestion
     * @param evaluate `true` if rules should be evaluated on select.
```
A readability improvement.